lib/queuePopulator/LogReader.js

```diff
@@ -136,7 +136,31 @@ class LogReader {
                         logOffset });
                 return done(null, logOffset);
             }
-            return done(null, 1);
+            return this._initializeLogOffset(done);
```

Comment:
Should this be specific to notifications or for all extensions using the `logReader`?

Reply:
I assume this should also be applicable replication!

Reply:
LGTM.